server/utils.py

```diff
@@ -194,3 +195,64 @@ def get_tale_files(tale, user):
     for item in artifact_items:
         files.append(get_file_item(item['_id'], user))
     return files
+
+
+def create_repository_file(recipe):
+    """
+    Creates a file that holds the recipe repository. Instead of
+     doing the collection,folder,item,file creation in the celery worker,
+     we'll do it here. We let the worker know which file to use by sending it
+     the file id returned from this function
+
+    :param recipe: The recipe whose repository is being saved
+    :type recipe: wholetale.models.recipe
+    :return: The file id or None
+    :rtype: str, NoneType
+    """
+
+    # Name for the folder & collection that will hold the repositories
+    folder_name = 'repository_bank'
+    folder_description = "Holds items that represent archived repositories"
+
+    """
+    Get all of the assetstores and retrieve GridFS. The reason we're using this
+     assetstore is that we don't want this folder in the user's home directory.
+    """
+    assetstores = ModelImporter.model('assetstore').list()
+    assetstore_id = int()
+
+    for store in assetstores:
+        if store['name'] == 'GridFS local':
```

Comment:
I don't expect that name to change. However, could we grab a list of assetstores by type and just pick the first one that's GridFS?

Reply:
Good call-let me check it out

Reply:
Committed